README.md

```diff
@@ -0,0 +1,152 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="" alt="Me&Ai logo" width="140"  height="auto" />
+  <br/>
```

Comment:
- [ ] Please fix the logo image path. The image is currently broken because the `src` attribute is empty. A broken logo image makes your repo unprofessional. YOu can either delete it / add your own logo. **Don't use the Microverse logo** 👍🏾